test/fixtures/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "threshold":     30,
+  "identifiers":   true,
+  "ignore":        "one", // used as RegExp,
```

Comment:
Since the `jsinspect` allows comments in JSON and uses `strip-json-comments` for it, that package was also added to this grunt plugin as a dependency